packages/react-composites/src/composites/CallWithChatComposite/components/PreparedDialpad.tsx

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+/* @conditional-compile-remove(people-pane-dropdown) */
```

Comment:
You should be able to remove *all* conditional compilation from this file.

Reply:
Need to conditional compile this and addpeopledropdown because both is using conditionally compiled components, if not will have an error in stable build